### ISSUE TITLE
[SampleCard-FlightDetails.json; Config-testVariantHostConfig.json]: The luminosity ratio of the label in name 'CHECK IN' and 'VIEW' buttons is not sufficient.

### ISSUE THREAD

**Issue by Bhaskar301101** [microsoft/AdaptiveCards#8479]
### Target Platforms

Android

### SDK Version

 2023.03.31.1

### Application Name

Adaptive Card

### Problem Description

Test Environment:
OS: Android 12
Device: S21+
Screen reader: Talkback
AdaptiveCard Android App version:  2023.03.31.1

Pre-requisite:
Import-Sample Cards->FlightDetails.json; Config-testVariantHostConfig.json

Repro Steps:

1. Navigate to the Visual.
2. Navigate to the sample card.
3. Check color contrast ratio for the 'CHECK IN' and 'VIEW' buttons
4. Observe the issue.

Actual Result:
The luminosity ratio of the label in name 'CHECK IN' and 'VIEW' buttons is 2.29:1 and 4.45:1 respectively.

Expected Result:
The luminosity ratio of the label in name 'CHECK IN' and 'VIEW' buttons should be atleast 4.5:1

Note:

1. This issue is also repro on below configs' for 'CHECK IN' and 'VIEW' buttons,
2. Default Config (CCA of 'VIEW' Button: 4.45:1)
3. Cortana-skills-classic.json (CCA of 'CHECK IN' button: 3.44:1, 'VIEW' Button: 4.45:1)
4. Cortana-skills-dark.json (CCA of 'CHECK IN' button: 1.82:1, 'VIEW' Button: 4.45:1)
5. Cortana-skills-light (CCA of 'VIEW' Button: 4.45:1)
6. Outlook-desktop.json (CCA of 'CHECK IN' button: 3.44:1, 'VIEW' Button: 4.45:1)
7. Render-to-image.json (CCA of 'CHECK IN' button: 3.44:1, 'VIEW' Button: 4.45:1)
8. Sample.json (CCA of 'VIEW' Button: 4.45:1)
9. Skype.json (CCA of 'CHECK IN' button: 3.44:1, 'VIEW' Button: 4.45:1)
10. Webchat.json (CCA of 'VIEW' Button: 4.45:1)
11. Windows-notification.json (CCA of 'CHECK IN' button: 3.44:1, 'VIEW' Button: 4.45:1)
12. Windows-tieline.json (CCA of 'CHECK IN' button: 3.44:1, 'VIEW' Button: 4.45:1)

User Experience:
Low vision users will might not able properly tell the text if color contrast is not sufficient.

WCAG Reference:
https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “,

### Screenshots

![MAS1 4 3 Check in issue](https://user-images.githubusercontent.com/89897257/232747327-8fb0ca64-42ba-42bc-87ff-33414691c7b4.jpg)
![MAS1 4 3 View issue](https://user-images.githubusercontent.com/89897257/232747331-4da60ef0-2ac1-421d-a60c-937e3ace7ec3.jpg)

### Card JSON

```JSON
NA
```

**Comment by jwoo-msft** [comment]
duplicate issue of hostconfig configuration options.
